pkg/resources/collector/templates.go

```diff
@@ -138,6 +139,14 @@ filelog/rke2:
     value: kubelet
 `))
 
+	templateTraceReceiver = `
```

Comment:
This configuration is static, so I'm not sure if it should be a template. I'm also unsure about how these templates are actually used in the code.

Reply:
You are correct you don't need a template here, since below you are already embedding the trace config into the template which gets translated to the yaml config the collector uses

Reply:
@jaehnri you will have to also [implement this for traces within this method](https://github.com/rancher/opni/blob/98ac856f97a4ecabf8ac5eff31c89d6c5a48c46d/plugins/logging/pkg/agent/drivers/kubernetes_manager/kubernetes_manager.go#L122-L139)